test/integration/ext/openai/model.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+import vcr
+
+from superduperdb.ext.openai.model import OpenAIChatCompletion, OpenAIEmbedding
+
+
+@vcr.use_cassette(
+    'test/integration/ext/openai/cassettes/test_embed.yaml',
+    filter_headers=['authorization'],
+)
+def test_embed():
+    e = OpenAIEmbedding(model='text-embedding-ada-002')
+    resp = e.predict('Hello, world!')
+
+    assert len(resp) == e.shape[0]
```

Comment:
Just to make it more informative, are there any boundaries we can add?  (Real quibble though.)

`assert 1 < len(resp) == e.shape[0] <= 24` or something.  (Chained conditions rock!)